.github/workflows/formatting.yaml

```diff
@@ -0,0 +1,19 @@
+name: Linting
+
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+  JavaFormat:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 # v2 minimum required
+      # Recommended: latest versions of Google Java Format require JDK 11+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "11"
```

Comment:
From the Maven config, it looks like this project is compiled with java-8:

```
<maven.compiler.source>1.8</maven.compiler.source>
<maven.compiler.target>1.8</maven.compiler.target>
```

Can we update this or the Maven config?

Reply:
Talked offline.